applications/ConvectionDiffusionApplication/tests/cpp_tests/test_laplacian_embedded_element.cpp

```diff
@@ -0,0 +1,123 @@
+// KRATOS ___ ___  _  ___   __   ___ ___ ___ ___
+//       / __/ _ \| \| \ \ / /__|   \_ _| __| __|
+//      | (_| (_) | .` |\ V /___| |) | || _|| _|
+//       \___\___/|_|\_| \_/    |___/___|_| |_|  APPLICATION
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//                   Franziska Wahl
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "includes/convection_diffusion_settings.h"
+#include "includes/variables.h"
+
+// Application includes
+#include "convection_diffusion_application.h"
+#include "convection_diffusion_application_variables.h"
+
+
+namespace Kratos {
+namespace Testing
+{
+    void SetLaplacianEmbeddedElementTestModelPart(ModelPart &rModelPart)
+    {
+        // Set buffer size
+        rModelPart.SetBufferSize(2);
```

Comment:
```suggestion
        rModelPart.SetBufferSize(1);
```
I think that 1 must be enough in this case.